guacamole/src/main/frontend/src/app/settings/services/preferenceService.js

```diff
@@ -142,6 +142,20 @@ angular.module('settings').provider('preferenceService', ['$injector',
          */
         language : getDefaultLanguageKey(),
         
+        /**
+         * The number of recent connections to display.
+         * 
+         * @type Integer
+         */
+        numberOfRecentConnections: 6,
+        
+        /**
+         * Whether or not to show the "Recent Connections" section.
+         * 
+         * @type Boolean
```

Comment:
This should be `@type {boolean}` (or `@type {!boolean}` if not nullable).

Reply:
Fixed via rebase.